src/fn/potentiometer.ts

```diff
@@ -0,0 +1,88 @@
+import { string, z } from "zod"
+import { platedhole } from "src/helpers/platedhole"
+import type { AnyCircuitElement, PcbSilkscreenPath } from "circuit-json"
+import { silkscreenRef, type SilkscreenRef } from "../helpers/silkscreenRef"
+
+export const potentiometer_def = z.object({
+  fn: z.string(),
+  num_pins: z.union([z.literal(3), z.literal(2)]).default(3),
+  p: z.string().default("3.8mm"),
+  id: z.string().default("1.25mm"),
+  od: z.string().default("2.35mm"),
+  ca: z.string().default("14mm"),
```

Comment:
Can you add a .describe? I dont know what ca is

Reply:
@techmannih just waiting on the .describe that shows what this is an abbreviation for